build.gradle

```diff
@@ -74,6 +74,16 @@ application {
     mainClass = 'com.epam.aidial.core.AiDial'
 }
 
+distributions {
+    // write current version to the version file
+    beforeEvaluate {
+        def versionFile = file("$projectDir/src/main/resources/version")
+        versionFile.getParentFile().mkdirs()
```

Comment:
what happens if resources or file already exists?

Reply:
according to the documentation, file and parent directory(s) will be created if not exists